short_to_long_pipeline/etl_pipeline.py

```diff
@@ -16,6 +17,11 @@
     "port": os.getenv("DB_PORT")
 }
 
+S3_BUCKET = os.getenv("S3_BUCKET")
+S3_KEY = os.getenv("S3_KEY", "plant_data/complete_data.parquet")
```

Comment:
It may be a good idea to give the file a more unique name, maybe through attaching the date. This will allow us to store all the files and differentiate them.